### ISSUE TITLE
menu et actions  définies coté serveur

### ISSUE THREAD

**Issue by denis-b** [SambaEdu/sambaedu-ipxe#1]
Pour l'auth, pour le moment ce n'est pas la peine de se prendre la tête, il faudra de toute façon le refaire pour AD... A mon avis il vaut mieux ce concentrer sur les actions à distance : pour les postes enregistrés, c'est le serveur qui autorise ou pas l'accès au menu  : il suffit d'enregistrer dans le ldap ou mysql le comportement du poste :

- menu verrouillé (o/n)
- action (boot win, boot linux, boot iscsi, action diverses, menu...)

En plus au niveau productivité c'est bien plus efficace de débloquer le menu pour un parc que de devoir se logguer sur chaque poste...

**Comment by denis-b** [comment]
Oui, cela peut être utile de proposer l'auth dans certains cas, mais j'ai du mal à voir lesquels : si qelqu'un arrive avec son portable perso et le fait booter en pxe, c'est généralement qu'il sait pourquoi il le fait. Il n'y a pas de raison de lui interdire. En revanche ensuite cela peut être intéressant de ne pas lui proposer les options d'installation avec mise au domaine.
 
Un autre truc qui pourrait être utile serait de pouvoir provisionner direct à la volée la machine dans un parc, en entrant le nom et le parc.

**Comment by mollef** [comment]
Les portables perso, pour moi, nous n'avons pas à les gérer ;). Je pensais plutôt à des actions manuelles à faire ponctuellement y compris sur une machine hors domaine. Par exemple dans une salle le poste est hs et je veux le restaurer. Je vais pas passer forcément par le serveur afin de débloquer le pxe mais intervenir en direct sur le poste, surtout s'il est seul dans la salle. Pour autant pouvoir débloquer sur un parc donné comme tu l'indiques a tout son sens. En conclusion pour moi les deux sont complémentaires.

**Comment by mbansse** [comment]
Dans mon lycée, je fais les sauvegardes locales avec l'interface se3, ainsi que par clonezilla (les deux sauvegardes cohabitent sur sda2). Du coup, quelques (rares) collègues qui travaillent toute la journée dans des salles informatiques ont le mdp du menu pxe, et lance une ligne spécialement faite pour eux pour relancer en 20 minutes un poste défaillant car mal éteint ou autre. Les deux sont donc effectivement complémentaires.
